packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx

```diff
@@ -111,15 +111,18 @@ const storybookMeta: Meta<TabsProps & TabItemProps> = {
       },
     },
     notificationCountMax: {
-      control: { type: "number" },
+      control: {
+        type: "select",
+      },
+      options: ["10", "20", "30", "40", "50", "60", "70", "80", "90", "100"],
       description:
-        "The limit at which the badge will show '`{notificationCountMax}`+'. A number between 0-999",
+        "The limit at which the badge will show `{notificationCountMax} +`. Can be increments of 10, up to 100",
       table: {
         type: {
-          summary: "number",
+          summary: "10 | 20 | 30 | 40 | 50 | 60 | 70 | 80 | 90 | 100",
```

Comment:
Curious why we decided on increments of 10 here — will some features really want the + to start at 20 vs 30?

Reply:
@jordankoschei-okta Mostly to try and enforce _some_ consistency. So folks don't set the max at something like `37` or whatever. We thought about just making it `10` but, I know for sure there is a use case that uses `20`. So, we just said increments of 10, up to 100 would probably work.